### PR TITLE
Add types for scroll

### PR DESCRIPTION
**Pull request by RomiC**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/scroll/tslint.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```
Comment (PranavSenthilnathan) [inline types/scroll/tslint.json]: Could you remove these rules?

File: types/scroll/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
+// Type definitions for scroll 3.0.1
```
Comment (PranavSenthilnathan) [inline types/scroll/index.d.ts]: This should be just `3.0` (one of the lint rules will catch this when it's reenabled).

**Review by PranavSenthilnathan — CHANGES_REQUESTED** [review]

File: types/scroll/index.d.ts (hunk 2 above)
Comment (RomiC) [reply types/scroll/index.d.ts]: So, am I right that that in case of minor changes type definitions should remain the same?

Comment (PranavSenthilnathan) [reply types/scroll/index.d.ts]: You can update the type definitions any time to keep up with patches/revisions of the underlying package as you wish. However, the idea is that patches shouldn't affect the type definitions that much (i.e. breaking changes should be major/minor version changes).

Comment (PranavSenthilnathan) [reply types/scroll/index.d.ts]: I might have misread your question - minor version changes in a package means that you should also rev up the minor version in the definition. So when 3.1.2 (for example) gets released, then you can update the definition to 3.1.

Comment (RomiC) [reply types/scroll/index.d.ts]: @PranavSenthilnathan 👍 Thank you for an explanation!

**Pull request merged**